extra/html-extra/CVA.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+/*
+ * This file is part of Twig.
+ *
+ * (c) Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Twig\Extra\Html;
+
+final class CVA
```

Comment:
(unfortunately), we decided in Symfony a long time ago to use `Cva` for such cases.